dashboard/application/config/rt_config.yml

```diff
@@ -0,0 +1,5 @@
+server: "http://rt:34000"
+user: "root"
+pass: "password"
+queues:
+  - "General"
```

Comment:
Do we need to add the "Alternate" queue in the local `rt_config.yml`?

Reply:
duplicate comment; see above